source/includes/_tag.md

```diff
@@ -1,6 +1,10 @@
 # Tag
 
-A tag is a label that may be attached to almost any resource. There are two categories of tags: `System Tags` and `User Tags`.
+A tag is a label that may be attached to almost any resource. While custom fields allow you to attach `{key, value}` pairs to various objects in the system, single values can also be
+attached to various objects in the system by using tags. There are two categories of tags: `System Tags` and `User Tags`. 
+
+* `System Tags`: These are interpreted by the system to change its behavior. Certain tags can only be attached to specific resource types -- e.g `Account`. In order to distinguish them from the user tags, the system tags are uppercase symbols.
+* `User Tags`: These are defined by the user for any desired purpose and are **not** interpreted by the system and can be anything as long as it a lowercase symbol. For example, `good_customer` could be a tag that can be attached to a customer `Account`. `User Tags` must be defined using the [Tag Definition](tag-definition.html) APIs. These tags may be attached to almost any resource type.
 
```

Comment:
The tags section already contains this info, so I do not think these changes are required.